x-pack/plugins/watcher/public/register_feature.js

```diff
@@ -12,7 +12,7 @@ import { i18n } from '@kbn/i18n';
 FeatureCatalogueRegistryProvider.register(() => {
   return {
     id: 'watcher',
-    title: 'Watcher',
+    title: i18n.translate('xpack.watcher.appTitle', { defaultMessage: 'Watcher' }),
```

Comment:
__note:__ I think we don't translate term `Watcher`, see other cases in https://github.com/elastic/kibana/blob/6.7/x-pack/plugins/translations/translations/zh-CN.json

Reply:
👍 Thanks! Do we have a guideline somewhere about which terms are translated and which are not?

Reply:
> Do we have a guideline somewhere about which terms are translated and which are not?

Yes, cc'ed you on #kibana-localization